backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/radiobutton/BpkRadioButton.kt

```diff
@@ -73,31 +71,33 @@ fun BpkRadioButton(
   interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
   content: @Composable RowScope.(Boolean) -> Unit,
 ) {
-  val rowModifier = if (onClick != null) {
-    modifier.selectable(
-      selected = selected,
-      interactionSource = interactionSource,
-      indication = null,
-      role = Role.RadioButton,
-      onClick = onClick,
-    )
-  } else {
-    modifier
-  }
+
   Row(
-    horizontalArrangement = Arrangement.spacedBy(BpkDimension.Spacing.Sm),
-    modifier = rowModifier,
     verticalAlignment = Alignment.CenterVertically,
+    horizontalArrangement = Arrangement.spacedBy(BpkDimension.Spacing.Sm),
+    modifier = modifier.applyIf(onClick != null) {
+      selectable(
+        selected = selected,
+        interactionSource = interactionSource,
+        indication = null,
+        role = Role.RadioButton,
+        onClick = onClick!!,
```

Comment:
Unfortunately the compiler cannot undertsand from the contract that onClick will never be null here